### DISCUSSION TITLE
Chunky death animations from old screenshots

### DISCUSSION THREAD

**Discussion by mefistotelis** [swfans/syndwarsfx#221] [General]
I started looking in more detail at the chunky death sprites from pre-alpha screenshots.

They seem to be specific to a person type - in dead zealots you can identify some of zealot features. The dying zealot could be at any angle, which would additionally multiply the amount of sprites by 6 - but that doesn't seem to be the case as the dead on all images have the same location of parts.
But still, additional 8 frames of death animation for every person model would add up to noticeable size - so I wonder if the switch to featureless puddle of blood animation was due to RAM constraints, rather than self-censoring.

![lvl-prealp01-img01b-best-pc_gamer-n021-1996_02-pg046-ss03-chunky-agent](https://github.com/user-attachments/assets/133c6900-3ec7-44f2-b66e-21590df5d9e7) ![lvl-prealp01-img03-best-secret_service-no032-1996_02-pg007-ss03-chunky](https://github.com/user-attachments/assets/cee60bd5-fba3-42ab-a06b-575156dac510) ![lvl-prealp01-img12-merge-pc_gamer-n021-1996_02-pg045-ss05-chunky](https://github.com/user-attachments/assets/dabaa52d-f5e8-4cc7-b5b5-6f05a6a50ba4) ![lvl-prealp15-merge-game_power-n047-pg031-ss05a-chunky1](https://github.com/user-attachments/assets/19b4098f-e49f-40e5-957a-03281713e718) ![lvl-prealp15-merge-game_power-n047-pg031-ss05a-chunky2](https://github.com/user-attachments/assets/2d2fc2f7-6ccb-4354-b301-a874fbdc6ef1) ![lvl-prealp15-merge-game_power-n047-pg031-ss05a-chunky3](https://github.com/user-attachments/assets/ec6a4c9a-bcd0-458a-a2ab-bd9a6d56c46d)
![lvl-prealp15-merge-game_power-n047-pg031-ss05a-chunky4](https://github.com/user-attachments/assets/6548170a-22ea-4026-afa9-1759702733c6) ![lvl-prealp15-merge-game_power-n047-pg031-ss05a-chunky5](https://github.com/user-attachments/assets/64d11eab-ec64-4c35-8a43-6cec240b6852) ![lvl-prealp15-merge-game_power-n047-pg031-ss05a-chunky6](https://github.com/user-attachments/assets/5716b196-1bd3-49eb-b692-cef6bc22cc2c) ![lvl-prealp15-merge-game_power-n047-pg031-ss05a-chunky8](https://github.com/user-attachments/assets/a2f1c398-a505-45c4-af92-f0cdee4bcf3e) ![lvl-prealp15-merge-game_power-n047-pg031-ss05a-chunky9](https://github.com/user-attachments/assets/594ec914-5314-4d2b-8fcd-ab7382773017) ![lvl-prealp15-merge-game_power-n047-pg031-ss05a-chunky10](https://github.com/user-attachments/assets/d057b08f-0e82-4be3-ac52-621ae94771ea) ![lvl-prealp15-merge-game_power-n047-pg031-ss05a-chunky11](https://github.com/user-attachments/assets/e527e3c0-d78d-49f1-aaab-e09fc47e6916)
![lvl-prealp19-img03-best-game_power-n047-pg032-ss01a-chunky1](https://github.com/user-attachments/assets/a86875b7-019a-469e-a354-d7a28a27b6ec) ![lvl-prealp19-img03-best-game_power-n047-pg032-ss01a-chunky3](https://github.com/user-attachments/assets/3c1a4797-dded-4db8-92c4-fe8f102dd157) ![lvl-prealp19-img04-merge-game_power-n047-pg032-ss03a-chunky1](https://github.com/user-attachments/assets/f620ea26-aed4-499c-bdf5-9bdebb25e3d7) ![lvl-prealp19-img04-merge-game_power-n047-pg032-ss03a-chunky2](https://github.com/user-attachments/assets/f7159e5e-01d9-4ec2-9e95-273c6e04f394)
![lvl-prealp19-img04-merge-game_power-n047-pg032-ss03a-chunky3](https://github.com/user-attachments/assets/8a9dc1b6-8c8f-4bf4-a6ea-4cd310c0c689) ![lvl-prealp19-img04-merge-game_power-n047-pg032-ss03a-chunky4](https://github.com/user-attachments/assets/2eeddaec-3f6f-40aa-aa9a-625703f3e28a) ![lvl-prealp19-img04-merge-game_power-n047-pg032-ss03a-chunky5](https://github.com/user-attachments/assets/350d3f26-773d-4a24-afa7-2a66052306b2)
![lvl-prealp19-img04-merge-game_power-n047-pg032-ss03a-chunky6](https://github.com/user-attachments/assets/2becfd59-521d-429e-83d4-cfc893075bf8) ![lvl-prealp19-img04-merge-game_power-n047-pg032-ss03a-chunky7](https://github.com/user-attachments/assets/8b3b5511-32f2-4b91-ae95-b005aadaaf44) ![lvl-prealp19-img04-merge-game_power-n047-pg032-ss03a-chunky8](https://github.com/user-attachments/assets/a6f6399e-a882-45cb-8075-5cb8a4986185) ![lvl-prealp19-img04-merge-game_power-n047-pg032-ss03a-chunky9](https://github.com/user-attachments/assets/65c2ef10-b0e1-484c-a5bb-7805a6610543) ![lvl-prealp19-img04-merge-game_power-n047-pg032-ss03a-chunky10](https://github.com/user-attachments/assets/0806e1b9-581c-4cf0-a535-5e19a2af056a) ![lvl-prealp19-img04-merge-game_power-n047-pg032-ss03a-chunky12](https://github.com/user-attachments/assets/0e5ad805-feef-48e8-a950-16b68fd18b22) ![lvl-prealp19-img04-merge-game_power-n047-pg032-ss03a-chunky14](https://github.com/user-attachments/assets/0ba460f3-5bee-400f-9d02-eb5d4888525f) ![lvl-prealp19-img05-merge-video_games-1995_07-pg017-ss01-chunky1](https://github.com/user-attachments/assets/d6b4739a-d9bb-4a8c-898d-7a9d55aa3c67) ![lvl-prealp19-img05-merge-video_games-1995_07-pg017-ss01-chunky3](https://github.com/user-attachments/assets/41282e83-5ac2-401c-ab1f-404aa11ac22b)

**Comment by mefistotelis** [comment]
Here are the death frames which the chunky animations would continue.
![pic0869](https://github.com/user-attachments/assets/561ce057-6cb8-4faa-9753-275aa2a7e87f) ![pic0873](https://github.com/user-attachments/assets/c6050159-87b2-4980-a003-066379106e17) ![pic0817](https://github.com/user-attachments/assets/14079317-5441-4b91-b761-963b251ba043)

**Reply by Moburma** [reply]
Ahh, good find. I dd notice them ages ago, but were not sure if they were used or not. I guess the question is where they removed due to being too gratuitous, or as you say, not enough RAM. The PS1 version is interesting in that respect, as the final game waste loads of space on unused junk. Hyperbole aside,, I think they genuinely might have had enough space for all the PC, but they kept all the unused original attempt at the PC's interface in RAM at all times in the PS1 version, even though unused. They still have at least 50% of the shady guy and scientist animations in RAM too, so if they had just cleaned those out they could probably have them full implemented. Maybe there was not room for the rest?

That's also why the Saturn version being leaked would be interesting, that would be a snapshot of the console versions before they were simplified.

**Comment by mefistotelis** [comment]
Here's the same set of zealot pics, cleaned up a bit:

![zealot-chunky-merge_0004_zealot-lvl-prealp19-img04-merge-game_power-n047-pg032-ss03a-chunky1](https://github.com/user-attachments/assets/a80d958d-cd01-4c40-9b27-ecca1a87c02b)
![zealot-chunky-merge_0005_zealot-lvl-prealp15-merge-game_power-n047-pg031-ss05a-chunky6](https://github.com/user-attachments/assets/ccf07891-479d-4232-a3b4-1979ffeaec19)
![zealot-chunky-merge_0006_zealot-lvl-prealp15-merge-game_power-n047-pg031-ss05a-chunky5](https://github.com/user-attachments/assets/9cb8ee15-f40c-499d-a84e-001801de800a)
![zealot-chunky-merge_0007_zealot-lvl-prealp15-merge-game_power-n047-pg031-ss05a-chunky4](https://github.com/user-attachments/assets/607e3d42-01a4-4898-a601-4546296c8b5f)
![zealot-chunky-merge_0008_zealot-lvl-prealp15-merge-game_power-n047-pg031-ss05a-chunky3](https://github.com/user-attachments/assets/e300a446-3126-4912-8903-600beec5c100)
![zealot-chunky-merge_0009_zealot-lvl-prealp15-merge-game_power-n047-pg031-ss05a-chunky2](https://github.com/user-attachments/assets/a7c387f8-0be5-469e-a716-e198c618bb1a)
![zealot-chunky-merge_0010_zealot-lvl-prealp15-merge-game_power-n047-pg031-ss05a-chunky1](https://github.com/user-attachments/assets/136bfb11-0502-4401-a99a-2af932b767ab)
![zealot-chunky-merge_0011_lvl-prealp19-img04-merge-game_power-n047-pg032-ss03a-chunky14 png](https://github.com/user-attachments/assets/c68847f8-c2bd-4138-9ac3-e7f0ad41d10d)
![zealot-chunky-merge_0000_zealot-lvl-prealp19-img04-merge-game_power-n047-pg032-ss03a-chunky12](https://github.com/user-attachments/assets/6005694d-7462-4ca3-81a2-7ed1b2d66dc0)
![zealot-chunky-merge_0001_zealot-lvl-prealp19-img04-merge-game_power-n047-pg032-ss03a-chunky8](https://github.com/user-attachments/assets/92335534-f0c4-4664-a23c-6c075acb81d4)
![zealot-chunky-merge_0002_zealot-lvl-prealp19-img04-merge-game_power-n047-pg032-ss03a-chunky4](https://github.com/user-attachments/assets/fb989209-04df-45a4-a2cb-b691210ca1ea)
![zealot-chunky-merge_0003_zealot-lvl-prealp19-img04-merge-game_power-n047-pg032-ss03a-chunky3](https://github.com/user-attachments/assets/0f32e5df-8374-467d-8869-d4283c99b92f)

**Reply by Moburma** [reply]
Hmm, I think we can probably recreate most of the above now, but the problem is all the between frames and the angles... Any artists lurking?

EDIT: Also it looks like they are actually the same graphic, just different palette/colouring depending on zealot or agent.

**Comment by mefistotelis** [comment]
> Also it looks like they are actually the same graphic, just different palette/colouring depending on zealot or agent.

No, I just added only the Zealot pics in that 2nd post.

EDIT:
Here are the agents:
![agent-chunky-merge_0000_agent-lvl-prealp19-img05-merge-video_games-1995_07-pg017-ss01-chunky3](https://github.com/user-attachments/assets/ed3bb696-900e-4751-9002-248afff604a4)
![agent-chunky-merge_0001_agent-lvl-prealp19-img05-merge-video_games-1995_07-pg017-ss01-chunky1](https://github.com/user-attachments/assets/4283f4a9-7456-49cf-b2c1-10afa1f8d707)
![agent-chunky-merge_0002_agent-lvl-prealp19-img04-merge-game_power-n047-pg032-ss03a-chunky14b](https://github.com/user-attachments/assets/0951a8c7-4c52-4a05-9da2-f288d7c63427)
![agent-chunky-merge_0003_agent-lvl-prealp19-img04-merge-game_power-n047-pg032-ss03a-chunky12b](https://github.com/user-attachments/assets/93e03cb0-2279-44f7-9999-8549ddb4d14f)
![agent-chunky-merge_0004_agent-lvl-prealp19-img04-merge-game_power-n047-pg032-ss03a-chunky10b](https://github.com/user-attachments/assets/76acf6fe-318a-4321-8ef5-c6817a7797ac)
![agent-chunky-merge_0005_agent-lvl-prealp19-img04-merge-game_power-n047-pg032-ss03a-chunky10](https://github.com/user-attachments/assets/00911616-5594-44f6-a50d-ffd39075ff40)
![agent-chunky-merge_0006_agent-lvl-prealp19-img04-merge-game_power-n047-pg032-ss03a-chunky9](https://github.com/user-attachments/assets/413f6058-f5e0-4bc6-a22c-55e130eca0c5)
![agent-chunky-merge_0007_agent-lvl-prealp19-img04-merge-game_power-n047-pg032-ss03a-chunky7](https://github.com/user-attachments/assets/ac6eb982-b471-4410-b9cd-a963b9f9d502)
![agent-chunky-merge_0008_agent-lvl-prealp19-img04-merge-game_power-n047-pg032-ss03a-chunky2](https://github.com/user-attachments/assets/538373a5-4fff-40b6-8010-d038c7a6fe99)
![agent-chunky-merge_0009_agent-lvl-prealp19-img03-best-game_power-n047-pg032-ss01a-chunky3](https://github.com/user-attachments/assets/262282bf-f7dd-4894-930a-0ce2eca7abb2)
![agent-chunky-merge_0010_agent-lvl-prealp19-img03-best-game_power-n047-pg032-ss01a-chunky1](https://github.com/user-attachments/assets/b6a9f17d-10ea-4280-8bb9-56e61b8130fa)
![agent-chunky-merge_0011_agent-lvl-prealp15-merge-game_power-n047-pg031-ss05a-chunky11](https://github.com/user-attachments/assets/3fd19a0c-a435-46f3-9eaa-79dab327a11f)
![agent-chunky-merge_0012_agent-lvl-prealp15-merge-game_power-n047-pg031-ss05a-chunky10](https://github.com/user-attachments/assets/b46dbcd9-f48a-4d96-acef-99270c1e5191)
![agent-chunky-merge_0013_agent-lvl-prealp15-merge-game_power-n047-pg031-ss05a-chunky9](https://github.com/user-attachments/assets/10c89308-eb91-477b-a75d-e5037d462cb1)
![agent-chunky-merge_0014_agent-lvl-prealp15-merge-game_power-n047-pg031-ss05a-chunky8](https://github.com/user-attachments/assets/f8f6b69f-5ac9-4573-ad12-588e7b74e75c)
![agent-chunky-merge_0015_agent-lvl-prealp01-img01b-best-pc_gamer-n021-1996_02-pg046-ss03-chunky-agent](https://github.com/user-attachments/assets/8f3ce006-8d04-4283-bb0c-680a1838a64e)
![agent-chunky-merge_0016_agent-lvl-prealp19-img04-merge-game_power-n047-pg032-ss03a-chunky5](https://github.com/user-attachments/assets/ca1e77e3-bd0d-405a-9ea8-8ce4f6420883)

**Comment by Moburma** [comment]
I had a go at recreating the zealot mess. After 10,000 hours in MS Paint I managed to come up with this so far:

![](https://i.ibb.co/KFgXGpL/guts1.png)

Some bits I've not even started on (i.e. top), and some are not right (right hand pauldron thing), but I think broadly getting there. The one thing I'm not sure on is the shading on the blood; the later games ones are dark red with paler edge lighting, but I get the feeling this might be more like the final exploding body one that is actually lighter in the middle and darker towards the edges. Also I dunno what the bit on the right above the head is, presumably a leg or something.

It struck me after attempting that the Agent one would be easier as a) it's just plain simpler with less to draw, and b) it has that tween frame which makes it clear which bit came from where. Maybe I will try that instead.

**Reply by mefistotelis** [reply]
The agent provides a very flat remains, I must say I don't really like its look. The zealot is perfect, but the agent looks like the bodies from Doom - the problem is that in Doom you look at them from a side, not from top.

But I was thinking about using these in different way - instead of the collection of remains, we could just add additional chunks to current animations.

ie. if a person dies from explosion, there's a chance of some bits (maybe just arms, or the shoulder armor in case of zealots) getting separated and landing few tiles further.

Then we would only need one animated body part (arm/shoulder), and the game would either use just one or two, one of which flipped, at different velocities.

Having separate things for the body parts would eliminate the problem with parts moving when camera is rotated.

EDIT:
Though we have to make sure it is gory rather than cartoonish. For example people kicking the remains while walking, or the remains bouncing during the fall, while visually interesting, wouldn't match the tone of the game.

**Reply by Moburma** [reply]
Yeah, one thing I realised when working on it is that the full zealot one must have been complex/elaborate, there's stuff all over the place and it's hard to follow where it came from. The agent one is much simpler and basically just gets ripped in half. The fact there only seem to have been one set of floor graphics despite starting frames for multiple angles probably explains what happened - they realised the sheer amount of effort needed for this for all characters and abandoned it for the generic end game "gibbed" animation shared by all. They did a similar thing in POP3, where the death animations for the different Shamans are actually an identical graphic and not the final sprites, if you look carefully.
